common/xslt/topnavigation.xsl

```diff
@@ -56,9 +56,12 @@
             </div>
             <div id="search">
                 <!-- append '+AND+idno%40type%3ADHQarticle-id' on form action to limit search to articles [CRB] -->
-                <form action="/dhq/findIt" method="get" onsubmit="javascript:document.location.href=cleanSearch(this.queryString.value); return false;">
-                    <div><input type="text" name="queryString" size="18" /><xsl:text> </xsl:text><input type="submit" value="Search" /></div>
-                </form>
+		    <form id="searchForm" method="get" action="/dhq/vol/search.html" enctype="application/x-www-form-urlencoded" accept-charset="utf-8">
+	<div id="search" title="Press the go! button to get to the advanced search page.">
+		<input id="q" type="text" name="q" value="" placeholder="search"/>

```

Comment:
Ummm … I _think_ I understand this, although it is a bit out of my usual toolbox. While I am a little uncomfortable with the word “keyword” as the placeholder text (because it is not a keyword search, but a full-text word search), I realize lots of other website use the word “keyword” in that same kinda lax way, so I suppose at least we would not be alone.
OK, done. But now the CSS needs to be changed to match, I suppose.